blog/_posts/2021-05-04-effective-programming-scala-course.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy
+title: "New Online Course: Effective Programming in Scala"
+---
+
+We are excited to announce that our new MOOC “Effective Programming in Scala” is now
+live on Coursera! We are also happy to share that most of our existing courses have already
+been updated to Scala 3.
+
+## Effective Programming in Scala
+
+The [Effective Programming in Scala] course
+is 6 weeks long, and aims to teach non-Scala programmers everything they need to become
+professional Scala 3 programmers. This course shows you how to embrace both functional
+programming and object-oriented programming in a simple and practical way to write software
+at scale. It explains how to achieve typical programming tasks in Scala,
+such as modeling business domains, breaking down complex problems into simpler problems,
+manipulating data, or running parallel tasks. Throughout the course, you will also learn
+the best practices for writing high-quality Scala code that scales to large applications,
+how to handle errors, how to write tests, and how to leverage a productive development
+environment.
+
+The course features six auto-graded programming assignments, which have been designed to
+be both fun and close to real-world programs. Topics covered by the assignments include
+graphic animations, web applications, property-based testing, generic programming,
```

Comment:
```suggestion
graphical animations, web applications, property-based testing, generic programming,
```
or
```suggestion
graph animations, web applications, property-based testing, generic programming,
```
depending on what you want to say.

"graphic" is an adjective that really does not mean the right thing, here.